jellyfin_kodi/objects/obj_map.json

```diff
@@ -13,6 +13,7 @@
 		"UniqueId": "ProviderIds/Imdb",
 		"Rating": "CommunityRating",
 		"Year": "ProductionYear",
+        "Premiere": "PremiereDate,ProductionYear",
```

Comment:
Shouldn't this only have one value?  (Haven't tested, so I don't know if it's actually invalid or not)

Reply:
Comma is fallback;

https://github.com/jellyfin/jellyfin-kodi/blob/4b948be28f3f11c62682c37a736df7bb35287f23/jellyfin_kodi/objects/obj.py#L41-L54